docs/command_line_export.md

```diff
@@ -1,6 +1,7 @@
 <a href="http://playeveryware.com"><img src="/docs/images/PlayEveryWareLogo.gif" alt="Lobby Screenshot" width="10%"/></a>
 
-# <div align="center">Command-line Export of Plugin</div><a name="documentation-style-guide" />
+# <div align="center">Command-line Export of Plugin</div>
+---
```

Comment:
This is really nitpicky, but now there are two lines below the main heading. One thin and one thick. I was ignoring it, but it does look messy so I'm calling it out here. It isn't just this file with it, they all have it so far.

![image](https://github.com/PlayEveryWare/eos_plugin_for_unity/assets/50968607/3cf6488c-e929-43d7-81b1-37439bd996ca)